### ISSUE TITLE
[high] Keyboard Navigation: Terms of service page: Additional links not focusable, appear only on hover

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5493]
## Action Performed:
1. Using Chrome, open https://staging.new.expensify.com/
2. Navigate to the 'terms of service' link using Tab key and press Enter key.
3. On 'Terms of service' page, press Tab key to navigate to the menu links from the left navigation pane.
4. Using mouse, hover over the menu links and observe the result.
5. Notice if the content is appearing when navigating using the keyboard or not.

## Expected Result:
When hovering over the 'Product', and 'Partners' links using mouse the additional links are revealed. It should be achieved when navigating using keyboard by pressing tab key as well.

## Actual Result:
When hovering over the 'Product', and 'Partners' links using mouse, the additional menu links are revealed. These additional links are not appearing when navigating using the keyboard by pressing tab key.

HTML:
`<li class=""><a href="/partners">Partners</a><ul class="subnav"><li class=""><a href="/accountants">Accountants</a></li><li class=""><a href="/banks">Banks</a></li></ul></li>`

## Workaround:
No workaround

## Area issue was found in:
Terms of Service page

## Failed WCAG checkpoints
2.1.1

## User impact:
If the content cannot be operated through a keyboard interface, it would not be operable by people with no vision. Some people with hand tremors as well as people who must use alternate keyboards or input devices that act as keyboard emulators find using a mouse very difficult. Individuals with low vision would also find it impossible to reach the menu links from the keyboard.

## Suggested resolution:
Ensure that the links which appear on hover are available for keyboard-only users as well. See: https://www.w3.org/WAI/tutorials/menus/flyout/#keyboard-users

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

https://user-images.githubusercontent.com/88733897/134706355-f7b49cad-dac9-40cd-b3c7-f16594efaa44.mp4

![Bug5180899_keyboard_-_terms_of_service](https://user-images.githubusercontent.com/88733897/134706364-ef08f11a-cb6e-4c38-b31b-1ed5c35afc93.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by shawnborton** [comment]
So the terms of service page is hosted on UseDot and it's technically not part of the NewDot app. @stitesExpensify @kadiealexander do you still think it's worth addressing these issues, or should we punt and focus on the app itself?

**Comment by stitesExpensify** [comment]
Yeah we will definitely not prioritize this one, but it should be fixed eventually IMO

**Comment by kadiealexander** [comment]
@RachCHopkins raised a really good point that our Terms of Service is what our users are agreeing to, and this should be completely navigable for any users with accessibility issues. I'm not sure if being able to access links inside the ToS is required for users, but I think we should definitely get it fixed.
